src/OneExplorer/OneExplorer.ts

```diff
@@ -470,31 +470,32 @@ input_path=${modelName}.${extName}
         readdirSyncRecursive(this.workspaceRoot!.fsPath).filter(val => val.endsWith('.cfg'));
 
     for (const conf of confs) {
-      const parsedObj = ConfigObj.parse(vscode.Uri.file(conf));
-      if (!parsedObj) {
+      const cfgObj = ConfigObj.createConfigObj(vscode.Uri.file(conf));
+
+      if (!cfgObj) {
         Logger.info('OneExplorer', `Failed to open file ${conf}`);
         continue;
       }
 
-      const {baseModels, derivedModels} = parsedObj;
+      if (!cfgObj.isChildOf(baseModelNode.path)) {
+        continue;
+      }
+
+      const {baseModels, derivedModels} = cfgObj.obj;
 
-      for (const baseModel of baseModels) {
-        if (this.comparePath(baseModel.fsPath, baseModelNode.path)) {
-          const pairNode = new Node(NodeType.config, [], vscode.Uri.file(conf));
-          Logger.debug('OneExplorer', `DerivedModels : ${derivedModels}`);
+      const pairNode = new Node(NodeType.config, [], vscode.Uri.file(conf));
+      Logger.debug('OneExplorer', `DerivedModels : ${derivedModels}`);
 
-          derivedModels ?.forEach(derivedModel => {
-                           // Display only the existing node
-                           const realPath = RealPath.createRealPath(derivedModel.fsPath);
-                           if (realPath) {
-                             pairNode.childNodes.push(new Node(
-                                 NodeType.derivedModel, [], vscode.Uri.file(realPath.absPath)));
-                           }
-                         });
+      derivedModels ?.forEach(derivedModel => {
+                       // Display only the existing node
+                       const realPath = RealPath.createRealPath(derivedModel.path!);
```

Comment:
(Q) Is this path always defined?

Reply:
Yes, the path is always defined by the point. (ditto)